Modules/xSharePoint/DSCResources/MSFT_xSPAppManagementServiceApp/MSFT_xSPAppManagementServiceApp.schema.mof

```diff
@@ -17,7 +17,7 @@ Database names or server name will not be changed if the configuration does not
 }
 */
 [ClassVersion("1.0.0.0"), FriendlyName("xSPAddInManagementServiceApp")]
```

Comment:
This is going to be one of your merge conflicts, we renamed it all before it got merged in mate. Plus here the friendly name doesn't match the rename done in this commit